step.sh

```diff
@@ -8,7 +8,8 @@ GEMFILE="--gemfile=${THIS_SCRIPT_DIR}/Gemfile"
 if [ "$CI" == "true" ]; then
     gem install bundler --force
     gem update --system
-    bundle update --bundler
+
+    bundle update --bundler --gemfile $GEMFILE
```

Comment:
According to line 5: `GEMFILE="--gemfile=${THIS_SCRIPT_DIR}/Gemfile"`

I guess `bundle update --bundler --gemfile --gemfile=...` would be an error.